### PR TITLE
log slot-uid

### PR DESCRIPTION
**Pull request by metachris**
## 📝 Summary

Instead of just sending it as part of the request, also log the slot uid with the getHeader/getPayload log messages. This should further help debugging.

---

## ✅ I have run these commands

* [x] `make lint`
* [x] `make test-race`
* [x] `go mod tidy`

### CODE REVIEW

File: server/service.go [hunk 1]
```diff
@@ -348,6 +339,16 @@ func (m *BoostService) handleGetHeader(w http.ResponseWriter, req *http.Request)
 	}
 	uid := m.slotUID.uid
 	m.slotUIDLock.Unlock()
+	log = log.WithField("slotUid", uid)
```
Comment (michaelneuder) [inline server/service.go]: nit: `slotUID` for consistency?

File: server/service.go [hunk 2]
```diff
@@ -614,13 +615,24 @@ func (m *BoostService) processCapellaPayload(w http.ResponseWriter, req *http.Re
...
+	m.slotUIDLock.Unlock()
+
 	// Prepare logger
 	ua := UserAgent(req.Header.Get("User-Agent"))
 	log = log.WithFields(logrus.Fields{
 		"ua":         ua,
 		"slot":       payload.Message.Slot,
 		"blockHash":  payload.Message.Body.ExecutionPayloadHeader.BlockHash.String(),
 		"parentHash": payload.Message.Body.ExecutionPayloadHeader.ParentHash.String(),
+		"slotUid":    uid,
```
Comment (michaelneuder) [inline server/service.go]: same comment as above

**Review by michaelneuder — DISMISSED** [review]

File: server/service.go (hunk 1 above)
Comment (metachris) [reply server/service.go]: done

File: server/service.go (hunk 2 above)
Comment (metachris) [reply server/service.go]: done

**Review by michaelneuder — APPROVED** [review]
lg

**Pull request merged**